orkui/template/default/Admin_player.tpl

```diff
@@ -25,6 +25,14 @@
 			<span></span>
 			<span><input type='submit' value='Update Media' name='Update' /></span>
 		</div>
+		<div>
+			<span></span>
+			<span style="font-size: 10px;">Supported Types: JPG, GIF, PNG</span>
+		</div>
+		<div>
+			<span></span>
+			<span style="font-size: 10px;">Max Size: 2725 Kb (348836 bytes)</span>
```

Comment:
Most people are not familiar with Kilobits and most file information within OSs is shown in KB or MB. I suggest changing to KB and omitting bytes all together.